docs/alerts.rst

```diff
@@ -0,0 +1,242 @@
+                    enriched: True
+         enriched_event_type: NETWORK
+           event_description: <truncated>
+                    event_id: <truncated>
+    event_network_inbound: False
+    event_network_local_ipv4: <truncated>
+    event_network_location: San Jose,CA,United States
+    event_network_protocol: TCP
+    event_network_remote_ipv4: <truncated>
+    event_network_remote_port: <truncated>
+       event_report_code: SUB_RPT_NONE
+      event_threat_score: [0]
+              event_type: netconn
+            ingress_time: 1632132315179
+                  legacy: True
+          netconn_domain: <truncated>
+         netconn_inbound: False
+            netconn_ipv4: <truncated>
+      netconn_local_ipv4: <truncated>
+      netconn_local_port: <truncated>
+        netconn_location: San Jose,CA,United States
+            netconn_port: <truncated>
+        netconn_protocol: PROTO_TCP
+                  org_id: <truncated>
+    parent_effective_reputation: LOCAL_WHITE
+    parent_effective_reputation_source: CERT
+             parent_guid: <truncated>-<truncated>-00000280-00000000-1d79a95c52...
+             parent_hash: ['<truncated>...
+             parent_name: c:\windows\system32\services.exe
+              parent_pid: 640
+       parent_reputation: NOT_LISTED
+         process_cmdline: ['C:\\Windows\\System32\\svchost.exe -k utcsvc ...
+    process_cmdline_length: [44]
+    process_effective_reputation: TRUSTED_WHITE_LIST
+    process_effective_reputation_source: APPROVED_DATABASE
+            process_guid: <truncated>-<truncated>-00000b44-00000000-1d79a95c67...
+            process_hash: ['<truncated>', '<truncated>...
+            process_name: c:\windows\system32\svchost.exe
+             process_pid: [2884]
+      process_reputation: ADAPTIVE_WHITE_LIST
+          process_sha256: <truncated>...
+      process_start_time: 2021-08-26T6:16:50.162Z
+        process_username: ['NT AUTHORITY\\SYSTEM']
+      triggered_alert_id: ['<truncated>-<truncated>-8af4-d6d0-e4bbe7917dff']
+                     ttp: ['PORTSCAN', 'MITRE_T1046_NETWORK_SERVICE_SCANN...
+    ...
+
+
+Watchlist Alerts
+----------------
+
+Process Details
+^^^^^^^^^^^^^^^
+
+You can retrieve each process details on each ``WatchlistAlert`` by using the example below. You can use list slicing
+to retrieve the first ``n`` results, in the example below ``10``. The ``get_details()`` method would give us metadata
+very similar to the one we've received by ``EnrichedEvent``.
+The full list of attributes and methods can be seen in the :py:mod:`Process() <cbc_sdk.platform.processes.Process>` class.
+
+.. code-block:: python
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.platform import WatchlistAlert, Process
+    >>> api = CBCloudAPI(profile='sample')
+    >>> alerts = api.select(WatchlistAlert)[:10]
+    >>> for alert in alerts:
+    ...     process = api.select(Process).where(process_guid=alert.original_document['process_guid']).first()
```

Comment:
Would it make sense to add a get_process helper function to the Watchlist alert model similar to what we did for the get_events on the CB Analytic alert?

Reply:
I think it should be added it will make it a lot cleaner. The code would simply be:
`api.select(Process).get_process(alert.original_document['process_guid'])` 

This will actually reduce the complexity of searching for a process and since it is an intensive task to look for processes, it will discourage the usage of the ORM syntaxis, where for example one might forget to type `.first()` and query all of the alerts.

Reply:
Oh yeah I've actually messed up the code I've meant to write `alert.get_process`, my bad.

Reply:
@dganev-cb How was this resolved as I don't see a new function added to the WatchlistAlert model? If that is going to be added later I would still recommend replacing the current process fetch with this code as this will fetch the specific device without needing to use the first() function
```
process = api.select(Process, alert.original_document['process_guid'])
```